code/game/gamemodes/gameticker.dm

```diff
@@ -700,13 +700,12 @@ var/datum/controller/gameticker/ticker
 	return text
 
 /datum/controller/gameticker/proc/achievement_declare_completion()
+	if((achievements.len == 0) || (achievements.len == null))
```

Comment:
```suggestion
	if(!achievements)
```

Reply:
i keep forgetting i can do this, ill fix it when i get back